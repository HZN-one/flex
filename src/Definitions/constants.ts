export const HZN_BASE_URL_ASSETS =
  process.env.HZN_BASE_URL_ASSETS ||
  "https://storage.googleapis.com/dev-hzn-assets";

export const DUMMY_IMAGE_URL =
  "https://dummyimage.com/250x180/e5e5e5/e5e5e5.jpg";

export const DUMMY_COMPANY_IMAGE_URL =
  "https://www.freepnglogos.com/uploads/netflix-logo-circle-png-5.png";

export const DEFAULT_IMAGE_INPUT_AVATAR = `${HZN_BASE_URL_ASSETS}/icons/inputAvatar.png`;
