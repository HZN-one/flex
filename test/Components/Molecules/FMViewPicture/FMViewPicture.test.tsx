/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { FMViewPicture } from "@Molecules";

import { render } from "@testing-library/react";

import { DUMMY_COMPANY_IMAGE_URL } from "@Definitions/constants";

describe("FMViewPicture", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FMViewPicture
        label="test"
        img={{ src: DUMMY_COMPANY_IMAGE_URL, alt: "dummy" }}
        testID="view-picture-test"
      />
    );
    expect(container).toMatchSnapshot();
  });
});
