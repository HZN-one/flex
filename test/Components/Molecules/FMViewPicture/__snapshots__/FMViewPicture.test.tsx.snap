// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMViewPicture should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-70qvj9"
    data-testid="view-picture-test"
  >
    <div
      class="MuiAvatar-root MuiAvatar-circular css-1fvb5lk-MuiAvatar-root"
      data-testid="avatar-profile-picture"
    >
      <img
        class="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
        src="https://storage.googleapis.com/dev-hzn-assets/icons/inputAvatar.png"
      />
    </div>
    <button
      class="MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeSmall MuiButton-outlinedSizeSmall MuiButtonBase-root  css-1kc3inn-MuiButtonBase-root-MuiButton-root"
      data-testid="button-view-logo"
      tabindex="0"
      type="button"
    >
      test
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
    <div
      aria-hidden="true"
      class="MuiBackdrop-root css-sox5kk-MuiBackdrop-root"
      data-testid="backdrop-image-picture"
      style="opacity: 0; visibility: hidden;"
    >
      <img
        alt="dummy"
        src="https://www.freepnglogos.com/uploads/netflix-logo-circle-png-5.png"
      />
    </div>
  </div>
</div>
`;
